array_analyzer/workflows/registration_workflow.py

```diff
@@ -144,13 +144,13 @@ def point_registration(input_folder, output_folder, debug=False):
         )
 
         # Estimate and remove background
+        im_crop = im_crop.astype('float64')
+        im_crop /= 255.0
```

Comment:
Instead of 255 I would recommend dividing by np.iinfo(im_crop.dtype).max just in case not all images are uint8.
And you don't have to convert to a float beforehand, the division will do that automatically.

Reply:
Ok we'll have to change it to
```
im_crop = im_crop/np.iinfo(im_crop.dtype).max
```

The problem I had with /= 255.0 is that numpy will complain if the two data types are different.  But I always thought the /= is faster?